### DISCUSSION TITLE
My account was suspended for no reason.

### DISCUSSION THREAD

**Discussion by thoxvi-support** [community/community#129145] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Today I suddenly cannot access my GitHub account: **@thoxvi**, and it shows that it has been suspended. Before this, I did not do anything against the rules of GitHub, and I haven't logged into GitHub for several days.

**Comment by thoxvi-support** [comment]
#2849433

**Comment by will-bvs** [comment]
Me too.
No response was received.

https://github.com/orgs/community/discussions/129048

**Comment by LiteBrite82** [comment]
Hi @thoxvi-support 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
